src/wp-includes/template.php

```diff
@@ -679,6 +679,19 @@ function get_attachment_template() {
 	return get_query_template( 'attachment', $templates );
 }
 
+/**
+ * Set up the globals used for template loading.
+ *
```

Comment:
```suggestion
 *
 * @since 6.5.0
 *
```

Missing `@since` annotation.

Reply:
Good catch. Holding off on adding this depending on if this gets backported to a 6.4.x minor release.

Reply:
Adding `@access private` would be good.

At the moment a 6.4.x before 6.5.0 doesn't look likely.

Reply:
For now, I've chosen not to mark this as private, given that it essentially replaces `wp_templating_constants()` which was also public.